### PR TITLE
New version: Nushell.Nushell version 0.76.0

### PR DESCRIPTION
**Pull request by fdncred**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97610)

### CODE REVIEW

File: manifests/n/Nushell/Nushell/0.76.0/Nushell.Nushell.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- nu
+ReleaseDate: 2023-02-21
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/nushell/nushell/releases/download/0.76.0/nu-0.76.0-x86_64-pc-windows-msvc.msi
+  InstallerSha256: b010a33f982ca2685677122cad651101f8c93016f59f07f54c88ae2024da9c44
```
Comment (sitiom) [inline manifests/n/Nushell/Nushell/0.76.0/Nushell.Nushell.installer.yaml:20]:
```suggestion
  InstallerSha256: B010A33F982CA2685677122CAD651101F8C93016F59F07F54C88AE2024DA9C44
```
Uppercase hash to match format of other manifests

**Review by sitiom — CHANGES_REQUESTED** [review]

File: manifests/n/Nushell/Nushell/0.76.0/Nushell.Nushell.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- nu
+ReleaseDate: 2023-02-21
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/nushell/nushell/releases/download/0.76.0/nu-0.76.0-x86_64-pc-windows-msvc.msi
+  InstallerSha256: B010A33F982CA2685677122CAD651101F8C93016F59F07F54C88AE2024DA9C44
+  ProductCode: '{6B693F7E-7721-4816-A4ED-12A651A10378}'
```
Comment (sitiom) [inline manifests/n/Nushell/Nushell/0.76.0/Nushell.Nushell.installer.yaml:21]:
```suggestion
  ProductCode: '{8EE1FEE1-80B8-47AC-99B4-39D1EED33CBE}'
```

Match new `ProductCode` as well

**Review by sitiom — CHANGES_REQUESTED** [review]

**Comment by fdncred** [comment]
I guess there's just no hope with getting this validation to work. Does anyone have any ideas?

**Comment by stephengillie** [comment]
Hi @fdncred,

This application asks to create a config file on first run. The waiver just applied will help it past this step, and hopefully complete validation.

**Review by stephengillie — APPROVED** [review]

**Comment by fdncred** [comment]
> Hi @fdncred,
> 
> This application asks to create a config file on first run.
> 
> > No environment config file found at
> > C:\Users\WDAGUtilityAccount\AppData\Roaming\nushell\env.nu
> > Would you like to create one with defaults (Y/n):
> 
> The waiver just applied will help it past this step, and hopefully complete validation.

Right, if there is no config file found, nushell asks if you want to create one. Is there a better way to handle this from a winget_create perspective?

**Pull request merged**